.github/workflows/publish.yaml

```diff
@@ -0,0 +1,17 @@
+name: Publish
+
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
Just `tags:` should be enough. And maybe only when `Cargo.toml` changes?

```suggestion
```

https://github.com/Traverse-Research/rust-template/blob/main/.github/workflows/publish.yaml

(That path needs to be changed for the workspace crates though)